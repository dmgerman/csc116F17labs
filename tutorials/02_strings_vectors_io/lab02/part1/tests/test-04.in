C++ is often considered to be a superset of C,
but this is not strictly true.
Most C code can easily be made to compile
correctly in C++, but there are a few differences
that cause some valid C code to be invalid or behave
differently in C++.
For example, C allows implicit conversion from void*
to other pointer types, but C++ does not (for type safety
reasons).
Also, C++ defines many new keywords, such as new and class,
which may be used as identifiers (for example, variable names)
in a C program.
