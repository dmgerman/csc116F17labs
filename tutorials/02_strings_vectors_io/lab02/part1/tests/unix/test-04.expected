,C fo tesrepus a eb ot deredisnoc netfo si ++C
.eurt yltcirts ton si siht tub
elipmoc ot edam eb ylisae nac edoc C tsoM
secnereffid wef a era ereht tub ,++C ni yltcerroc
evaheb ro dilavni eb ot edoc C dilav emos esuac taht
.++C ni yltnereffid
*diov morf noisrevnoc ticilpmi swolla C ,elpmaxe roF
ytefas epyt rof( ton seod ++C tub ,sepyt retniop rehto ot
.)snosaer
,ssalc dna wen sa hcus ,sdrowyek wen ynam senifed ++C ,oslA
)seman elbairav ,elpmaxe rof( sreifitnedi sa desu eb yam hcihw
.margorp C a ni
